runestone/common/project_template/pavement.tmpl

```diff
@@ -34,8 +34,6 @@ options(
         outdir=serving_dir,
         confdir=".",
         template_args={
-            'login_required': '%(login_req)s',
```

Comment:
I'm not in favor of removing `login_required` This is still a very common use case for other individuals hosting runestone books on a simple server.  AND runestone.academy must still support the use of non-login books as base courses.  Every book on academy must have a free to read without login version.

Reply:
Effectively, `login_required` is a duplicate of `useRunestoneServices` -- we should eliminate one!

Reply:
I don't think they are duplicates.  Today, we still use Runestone services even if not logged in.  Actions are just logged anonymously.